app/src/test/java/org/pesmypetcare/mypetcare/services/StubAchievementService.java

```diff
@@ -0,0 +1,43 @@
+package org.pesmypetcare.mypetcare.services;
+
+import org.pesmypetcare.mypetcare.features.users.User;
+import org.pesmypetcare.mypetcare.services.achievement.AchievementService;
+import org.pesmypetcare.usermanager.datacontainers.user.UserMedalData;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.TreeMap;
+
+/**
+ * @author Daniel Clemente
+ */
+public class StubAchievementService implements AchievementService {
+    private Map<String, List<UserMedalData>> medals;
+
+    public StubAchievementService() {
+        medals = new TreeMap<>();
+        UserMedalData userMedalData = new UserMedalData("Medal", Arrays.asList(10.0, 20.0, 30.0), "Test", new byte[0],
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['10.0' is a magic number.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5653081)